### DISCUSSION TITLE
Option to confirm save changes back to the currently connected database

### DISCUSSION THREAD

**Discussion by GrimnebulinAhab** [TabularEditor/TabularEditor3#921] [Ideas]
It would be nice to have an option confirm changes before automatically saving back to the connected database.

**Comment by otykier** [comment]
That should indeed be an option, as a confirmation dialog upon every save would probably feel too intrusive when working in a development environment or with a local source control repository.

**Reply by GrimnebulinAhab** [reply]
Totally agree. It would just be nice to have that option. I prefer using keyboard shortcuts, and with having multiple monitors, one can run the risk of trying to save in another program and accidentally saving back to the database because the wrong window was in focus. Thanks for the quick reply!
